CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - GDoc Importer: Support `[no-caption]` text, returns empty caption for element
 - Fix AMP export of Twitter tweets
 - Add a plain text exporter
+- Support multiple calls to `Article#place_additional_elements`
```

Comment:
It was already supported before, the results were just bad :)
So maybe
> Improve placement for multiple calls to `Article#place_additional_elements`
?

Reply:
I agree..

Reply:
Good point, changing it :)